### ISSUE TITLE
[Bug] Está enviando e-mail ao alterar pedido. Após clicar em confirmar alterações, mesmo marcado para não enviar.

### ISSUE THREAD

**Issue by rlasmar** [opencartbrasil/opencartbrasil#33]
**Em que versão do OpenCart Brasil o erro ocorre?**
1.4.6

**Que parte da loja é afetada pelo erro?**
Admin - Pedidos - Editar algum pedido - Última tela - Ao clicar em salvar.
Está enviando e-mail, quando não deveria enviar.

**Faça um resumo do problema ocasionado pelo erro:**
1. Ao editar um pedido, seja adicionando novo produto, alterando endereço, etc, na última tela, quando clica em salvar, ele está enviando e-mail para o comprador, com as informações que foram alteradas.
No entanto, no histórico do pedido, ele atualiza o status corretamente, porém está marcado para não informar o cliente, mas mesmo assim o envio do e-mail ocorre.

**Link para imagens com capturas da tela de erro:**
1 Tela de edição do pedido, ao clicar em salvar.
![1](https://user-images.githubusercontent.com/37814975/70563041-e8394100-1b63-11ea-8425-1e096590be81.JPG)

2. No histórico do pedido, mostra cliente notificado: NÃO
Porém, o e-mail está sendo enviado, já mostrando os dados que foram alterados.
![2](https://user-images.githubusercontent.com/37814975/70563283-5a118a80-1b64-11ea-95ec-1e91c4c9d51d.JPG)

**Comment by rlasmar** [comment]
Talvez uma opção melhor seria na tela 1 onde edita os pedidos, ter uma opção de escolher notificar ou não.

**Comment by v5mvidal** [comment]
Não se trata de um bug, mas de um comportamento padrão do OpenCart, por isso marquei como sugestão.

**Comment by rlasmar** [comment]
Olá Manoel,

Mesmo quando no histórico do pedido aparece para não ter sido notificado, como mostra na imagem 2?

**Comment by v5mvidal** [comment]
Sim, porque não tem haver com o histórico do pedido, mas sim com o envio de um e-mail para o cliente informando os detalhes do pedido após as alterações.
